### PR TITLE
{bio} [foss,2015b] Subread 1.5.0-p1 (REVIEW)

### PR DESCRIPTION
**Pull request by verdurin**
Subread 1.5.0-p1, another bioinformatics package

### CODE REVIEW

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
maraba - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz, Python 2.7.11
See https://gist.github.com/f24fa73cfc43e886e4532d8e388ed768 for a full test report.

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/s/Subread/Subread-1.5.0p1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+homepage = 'http://subread.sourceforge.net/'
+description = """High performance read alignment, quantification and mutation discovery"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+source_urls = [('https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s-%(version)s', 'download')]
+
+prebuildopts = 'cd src;'
```
Comment (boegel) [inline easybuild/easyconfigs/s/Subread/Subread-1.5.0p1-foss-2016a.eb]: use `start_dir = 'src'`

effect should be the same though

if you prefer this, then replace `; ` with `&&` (to chain the commands together such that one subcommand failing results in cancelling the rest)

File: easybuild/easyconfigs/s/Subread/Subread-1.5.0p1-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+source_urls = [('https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s-%(version)s', 'download')]
+
+prebuildopts = 'cd src;'
+buildopts = '-f Makefile.Linux'
+
+files_to_copy = ['bin', 'annotation', 'doc', 'LICENSE', 'README.txt', 'src', 'test']
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/s/Subread/Subread-1.5.0p1-foss-2015b.eb]: drop duplicate empty line please

**Comment by boegel** [comment]
@verdurin please take a look at the style remarks (which are now hidden by the filename fix you added)

**Comment by verdurin** [comment]
@boegel - should be fixed now

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cd8aa5bbc0ff256c61d13077c25a4eeb for a full test report.

**Comment by boegel** [comment]
@verdurin hmm, seems like we broke it... :(

The `src` directory is probably one level down?
If you specify `start_dir`, you need to specify the entire path relative to the unpacked sources.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a8a84761cff0252dbc55fdb286d8ec1e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**